tests/runtime_neutral/test_current_routing_contract_scan.py

```
@@ -1,84 +1,125 @@
+        if candidate and Path(candidate).exists():
+            return str(Path(candidate))
+    return None
+
+
+class CurrentRoutingContractScanTests(unittest.TestCase):
+    def test_scan_script_reports_json_and_no_current_surface_violations(self) -> None:
+        shell = resolve_powershell()
+        if shell is None:
+            self.skipTest("PowerShell executable not available")
+
+        completed = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)